templates/user/admin.html.twig

```diff
@@ -0,0 +1,25 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Gestion des administrateurs{% endblock %}
+
+{% block body %}
+    <div class="container">
+        <h1>Gestion des administrateurs</h1>
+        {% for flashMessage in app.flashes('success') %}
```

Comment:
Tu n'es pas obligé de faire 2 blocs : 
https://symfony.com/blog/new-in-symfony-3-3-improved-flash-messages